.ci/regression-report.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
copy paste is too big.
![image](https://github.com/user-attachments/assets/30b28a51-a395-4464-a9f8-e5bbd78b69bf)

can we update diff-report.yml to expect different file name ?

Reply:
let copy from web file as is.

in  bash we will just get fist by extension (there supposed to be only one property file):
```
property_files=(*.properties)
PROPERTY_FILE=$({files[0]})

```

Reply:
please create separate PR to update old workflow, I understand that testing is complicated, I think we can merge this without pre-testing, we will test after merge.

Reply:
Workaround at https://github.com/checkstyle/checkstyle/pull/15759

Resolved.

I will keep it "unresolved" in GitHub just to let others see reason easier